CTP/shell/src/com/navercorp/cubridqa/shell/common/CommonUtils.java

```diff
@@ -102,6 +116,54 @@ public static String rightTrim(String str) {
 		
 	}
 	
+	public static String getSimplifiedBuildId(String cubridPackageUrl) {
```

Comment:
It's better to use these in common packages.
com.navercorp.cubridqa.common.CommonUtils.getBuildId(String)
com.navercorp.cubridqa.common.CommonUtils.getSimplifiedBuildId(String)
com.navercorp.cubridqa.common.CommonUtils.getBuildBits(String)